service/compute/api.go

```diff
@@ -233,6 +233,114 @@ func (a *ClustersAPI) Impl() ClustersService {
 	return a.impl
 }
 
+// WaitGetClusterRunning calls [ClustersAPI.Edit] and waits to reach RUNNING state
+func (a *ClustersAPI) WaitGetClusterRunning(ctx context.Context, clusterId string,
```

Comment:
This name is a bit clunky. I would have expected `GetClusterWaitRunning`.

Reply:
@pietern it's a common name across all SDKs, all with common `Wait` prefix.

Reply:
Alright, works.